pkg/plugins/golang/v4/scaffolds/internal/templates/api/webhook.go

```diff
@@ -97,16 +101,27 @@ import (
 	{{- if .Resource.HasValidationWebhook }}
 	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
 	{{- end }}
+	{{- if .ExternalAPI }}
+	{{ .Resource.ImportAlias }} "{{ .Resource.Path }}"
+	{{- end }}
 )
 
 // nolint:unused
 // log is for logging in this package.
 var {{ lower .Resource.Kind }}log = logf.Log.WithName("{{ lower .Resource.Kind }}-resource")
 
 // SetupWebhookWithManager will setup the manager to manage the webhooks.
+{{- if .ExternalAPI }}
+func SetupWebhookFor{{ .Resource.Kind }}WithManager(mgr ctrl.Manager) error {
```

Comment:
Those changes will not be required after : https://github.com/kubernetes-sigs/kubebuilder/pull/4150